### PR TITLE
Fix: Report Export Types

### LINKED ISSUE #1995: MS Cache > Diccionario > Reportes
## Descripción
Se requiere implementación de un caché genérico para mejorar el rendimiento y la eficiencia del sistema, es necesario implementar un caché para los **Reportes** en el **Diccionario**.

### Objetivo
El objetivo de este issue es diseñar e implementar un mecanismo de caché para los Reportes en el Diccionario. Esto permitirá acceder a los datos de los reportes de manera más rápida y eficiente

### LINKED ISSUE #2068: [Bug Report] Error Multiples Reportes
## Bug report
No se puede previsualización mas de un reporte a la ves
#### Steps to reproduce

1. ejecute el reporte de Saldos Abiertos
2. previsualización el reporte 
3. ejecute el reporte de Saldos de Socio del Negocio
4. Observe el error
#### Screenshot or Gif（截图或动态图）

https://github.com/solop-develop/frontend-core/assets/45974454/4c66cea3-821b-46fd-b216-27173fe955e4

### PR DESCRIPTION
**Pull request by elsiosanchez**
#### Screenshot or Gif

https://github.com/solop-develop/frontend-core/assets/45974454/8985c8c0-8b6e-4832-a214-7414ef3a30ee

#### Additional context

fixes https://github.com/solop-develop/frontend-core/issues/1995

### CODE REVIEW

File: src/utils/ADempiere/dictionary/process.js [hunk 1]
```diff
@@ -151,6 +151,7 @@ export function generateProcess({
 
     fieldsList = fieldsList.concat(fieldsRangeList)
     // order range fields
+    console.log({ fieldsList })
```
Comment (EdwinBetanc0urt) [inline src/utils/ADempiere/dictionary/process.js]: @elsiosanchez please remove this console.log

File: src/utils/ADempiere/constants/report.js [hunk 2]
```diff
@@ -32,3 +34,56 @@ export const staticReportRoutes = [
     parametersList: [{ columnName: 'ValidFrom', value: today }, { columnName: 'MustBeStocked', value: false }]
   }
 ]
+
+export const reportExportTypes = [
```
Comment (EdwinBetanc0urt) [inline src/utils/ADempiere/constants/report.js]: Rename the constant to upper case `REPORT_EXPORT_TYPES`

**Review by EdwinBetanc0urt — APPROVED** [review]

File: src/utils/ADempiere/constants/report.js (hunk 2 above)
Comment (elsiosanchez) [reply src/utils/ADempiere/constants/report.js]: [95ebee3](https://github.com/solop-develop/frontend-core/pull/2060/commits/95ebee31ba8310075bb2f93da62a0008deccb6ab)

File: src/utils/ADempiere/dictionary/process.js (hunk 1 above)
Comment (elsiosanchez) [reply src/utils/ADempiere/dictionary/process.js]: [95ebee3](https://github.com/solop-develop/frontend-core/pull/2060/commits/95ebee31ba8310075bb2f93da62a0008deccb6ab)

**Review by EdwinBetanc0urt — APPROVED** [review]

**Pull request merged**